orderer/common/channelparticipation/restapi.go

```diff
@@ -176,14 +178,14 @@ func (h *HTTPHandler) serveJoin(resp http.ResponseWriter, req *http.Request) {
 	}
 
 	info, err := h.registrar.JoinChannel(channelID, block, isAppChannel)
-	if err == nil {
-		info.URL = path.Join(URLBaseV1Channels, info.Name)
-		h.logger.Debugf("Successfully joined channel: %s", info)
-		h.sendResponseCreated(resp, info.URL, info)
+	if err != nil {
```

Comment:
Minor refactor here to invert the logic for consistency with the rest of this function and file.